package.json

```diff
@@ -57,7 +60,7 @@
   },
   "style": "dist/css/bootstrap.css",
   "sass": "scss/bootstrap.scss",
-  "main": "dist/js/bootstrap",
+  "main": "src/js/index",
```

Comment:
Re: `ends up loading ALL modules`  issue is documented in my original issue https://github.com/twbs/bootstrap/issues/19017. Re-exported modules will call all the IIFEs when the index file is evaluated, leading into namespace pollution.  See https://codesandbox.io/s/vLXO14lg for an example.

In my [branch](https://github.com/petetnt/bootstrap/commits/v4-js-modules) I opted in for having module users to initialize their own components with `ModuleNameHere.init()`, while the browser bundles (including single modules, like tooltip.js) auto-initialize themselves with a secret variable ([[1]](https://github.com/petetnt/bootstrap/blob/v4-js-modules/js/src/index.js) [[2]](https://github.com/petetnt/bootstrap/blob/v4-js-modules/js/webpack.config.js#L49-L54)) which solves the problem but creates some cognitive load for the end user.